src/library/Uno.Themes.WinUI.Markup/Theme/CalendarDatePicker.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup;
+using Uno.Extensions.Markup.Internals;
+
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static partial class CalendarDatePicker
+		{
+			public static partial class Resources
+			{
+				public static readonly BackgroundVSG Background = new();
+				public partial class BackgroundVSG
```

Comment:
Now that I see it more, the `VSG` bothers me because it'll be a public class and I feel people will be confused why Resources.Backgound is of type `BackgroundVSG` :/ But I guess there's no way around it. 

Maybe `BackgroundGroup` or `BackgroundGrouping` or `BackgroundResourceGroup`? 

`BackgoundBrushGroup`? Although I guess it's not always restricted to just Brushes when we group them by visual state variant

`BackgroundVariants`??

Reply:
Yeh the naming of this bugs me too. Given the purpose of the BackgroundVSG is actually to house the different states of the background (it isn't really a mapping to the visual state group), we can just call it `BackgroundStates`